src/main/java/io/crate/client/jdbc/CrateDriver.java

```diff
@@ -56,9 +56,19 @@ public Connection connect(String url, Properties info) throws SQLException {
             throw new SQLException(String.format("Protocol url %s not supported. Must be one of %s or %s", parts[0]+"//", PREFIX, LONG_PREFIX));
         }
 
-        CrateConnection connection = new CrateConnection(new CrateClient(url.split(",")), url);
-        connection.connect();
+        CrateConnection connection;
+        if (url.equals("/")) {
+            connection = new CrateConnection(new CrateClient(), url);
+        } else {
+            String[] urlParts = url.split("/");
+            String hosts = urlParts[0];
+            connection = new CrateConnection(new CrateClient(hosts.split(",")), url);
+            if (urlParts.length == 2) {
+                connection.setSchema(urlParts[1]);
+            }
```

Comment:
handle invalid urls with three parts for example, and add some tests for those cases